Homework_3/task_4.py

```diff
@@ -0,0 +1,18 @@
+#Создайте словарь со списком вещей для похода в качестве ключа и их массой в качестве значения. 
+#Определите какие вещи влезут в рюкзак передав его максимальную грузоподъёмность. 
+#Достаточно вернуть один допустимый вариант. 
+#*Верните все возможные варианты комплектации рюкзака.
+
+items = {'Кружка' : 3, 'Ложка' : 1, 'Миска' : 6, 'Туалетная бумага' : 2}
+capacity = 7
+count = 0
+pack = []
+
+for i in items:
+    if count + items.get(i) <= 7:
```

Comment:
```suggestion
    if count + items.get(i) <= capacity :
```
А вот тут, кстати, метод get используется не совсем верно. Обычное обращение по ключу было бы более правильным.